ros2_ws/src/ur_moveit_demo/CMakeLists.txt

```diff
@@ -14,6 +14,7 @@ find_package(control_msgs REQUIRED)
 find_package(tf2_msgs REQUIRED)
```

Comment:
Please document the package in the demo README. Refer to the O3DE documentation where applicable.